modules/@apostrophecms/doc-type/index.js

```diff
@@ -1067,6 +1060,14 @@ module.exports = {
         const existing = await actionModule.findForEditing(toReq, {
           _id: toId
         }).permission('view').toObject();
+
+        const eventOptions = {
+          source: draft.aposLocale.split(':')[0],
+          target: toLocale,
+          docType: draft.type,
```

Comment:
The `type` property is not present in the passed document (since it keeps only properties present in the schema..)